package.json

```diff
@@ -100,7 +101,7 @@
     "prettier": "2.5.1"
   },
   "lint-staged": {
-    "**/*.js?(x)": [
+    "**/*.{js, jsx, ts, tsx}": [
```

Comment:
does it work okay with spaces? afaik native sh/bash won't allow it. probably it's different syntax here, just wondering

Reply:
It makes sense to up remove the spaces. I will update to `src/**/*.{js,ts,jsx,tsx}`